care/utils/serializer/external_id_field.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import serializers
+from rest_framework.fields import empty
+
+
+class UUIDValidator(object):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'UUIDValidator' inherits from object, can be safely removed from bases in python3](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5498155)